README.md

```diff
@@ -13,17 +13,18 @@ adhocracy+ is designed to make online participation easy and accessible to every
 
 ### Requirements
 
```

Comment:
Since we are revisiting the README, we can also add the tech docs.
On line 3 please add a link to our technical documentation.
E.g The project's [technical documentation](https://liqd.github.io/adhocracy-plus/) is in progress. You are welcome to leave your feedback on the documentation by creating a github issue.